.travis.yml

```diff
@@ -23,7 +23,7 @@ git:
 env:
   - TRAVIS_TESTS="unix docs translations" TRAVIS_BOARDS="feather_huzzah circuitplayground_express pca10056 pca10059 feather_nrf52832 feather_nrf52840_express" TRAVIS_SDK=arm:nrf:esp8266
   - TRAVIS_BOARDS="metro_m0_express metro_m4_express pirkey_m0 trellis_m4_express trinket_m0" TRAVIS_SDK=arm
-  - TRAVIS_BOARDS="feather_radiofruit_zigbee gemma_m0 hallowing_m0_express itsybitsy_m0_express itsybitsy_m4_express meowmeow" TRAVIS_SDK=arm
+  - TRAVIS_BOARDS="feather_radiofruit_zigbee gemma_m0 hallowing_m0_express itsybitsy_m0_express itsybitsy_m4_express meowmeow arduino_mkr1300" TRAVIS_SDK=arm
```

Comment:
We're trying to keep the number of boards listed on each line roughly the same, so the travis subjobs will finish in about the same amount of time. Could you move the `arduino_mkr1300` entry down into the line with `arduino_zero`?